resources/views/newDesign/news/includeNews/newsPage.blade.php

```diff
@@ -20,10 +20,18 @@
             <div id="left-content-column" class="col-xs-9">
                 <h3 class="nameNews">{{ $newsOne->name}}</h3>
                 @if($newsOne->img!='Not picture')
-                    <div><img class="picture" src="{{ asset($newsOne->getPath().$newsOne->img) }}" ></div>
+                    <div><img class="picture" src="{{ asset($newsOne->getPath().$newsOne->img) }}"></div>
                 @endif
                 <p>{!! $newsOne->description !!}</p>
                 <span><h4>Опубліковано </h4>{{date('j.m.Y', strtotime($newsOne->updated_at))}}</span>
+                <p>
+                    @if($newsOne->id > 1)
+                        {!! Html::linkRoute('news.show', '&laquo; Previous',  ['id' => $newsOne->id - 1],[ 'class' => 'previous pull-left']) !!}
```

Comment:
It's isn't good decision
You should find next news in controller and don't find next/previous news via -+ id.
Because admin can destroy some news and id doesn't be valid